tests/unit/app/test_sentry.py

```diff
@@ -0,0 +1,19 @@
+"""
+Module for testing Sentry integration
+"""
+# pylint: disable=cannot-enumerate-pytest-fixtures
+
+import json
+import os.path
+from unittest.mock import patch
+
+
+def test_errors_are_reported_to_sentry(anon_client):
+    with patch("src.app.api.sentry_sdk.capture_exception") as mocked:
+        with patch("src.jbi.router.execute_request", side_effect=ValueError):
+            try:
+                anon_client.post("/bugzilla_webhook")
+            except Exception:
+                pass  # This shouldn't be necessary, the API should return 500.
```

Comment:
@bsieber-mozilla Is it expected that the server will crash on exceptions? I guess that in that case, Gunicorn (or Nginx) will be in charge of returning the 500 response. Just wanted to confirm :)

Reply:
On exceptions there should be a 500 response without crashing the server; I checked out the branch to test this and confirmed.

Reply:
I don't say that the server crashes. But the application does. 
Gunicorn catches the crash and returns a 500. 

It's totally acceptable, we just have to take it into account in unit tests, since we don't test the app through gunicorn